app/onboarding/state/page.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
+
+import { useProfile } from '@/hooks/useProfile';
+import {
+  Button,
+  ButtonContainer,
+  ContentContainer,
+  PageContainer,
+} from '../styles';
+
+export default function Page() {
+  const { addProfile } = useProfile();
+
+  const handleButtonClick = (state: string) => {
+    const newProfile = { state, email: '', phone_num: '', user_type: '' };
```

Comment:
it might be cleaner to type this as Partial<Profile>, i.e.

`const newProfile: Partial<Profile> = { state };`
For more info on the Partial type: https://dev.to/martinpersson/better-code-quality-with-typescripts-utility-types-pick-partial-and-omit-3605